plugin.video.manoto/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.manoto"
        name="Manoto TV"
-       version="1.2.2"
+       version="1.2.3"
        provider-name="babak">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Please change xbmc.python to 2.14.0 for gotham. Please see http://kodi.wiki/view/Addon.xml